c7n/resources/cloudfront.py

```diff
@@ -149,6 +160,37 @@ def process(self, resources, event=None):
         return results
 
 
+@Distribution.filter_registry.register('logging-enabled')
+class IsLoggingEnabled(Filter):
+    """Check if logging is enabled for a Cloudfront distribution.
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: logging-enabled
+                resource: distribution
+                filters:
+                  - type: logging-enabled
+   """
+
+    schema = type_schema('logging-enabled')
+
+    def process(self, resources, event=None):
+        client = local_session(self.manager.session_factory).client(
+            'cloudfront')
+
+        results = []
+        for r in resources:
+            result = client.get_distribution_config(Id=r['Id'])
```

Comment:
Thanks for the pr,  preference would be for the generic filter on config via valuefilter subclass with annotation of the lookup result rather than fetching the info for a specific check and discarding the rest

Reply:
Can you provide an example of what you were thinking as a better alternative? 

Thanks

Reply:
basically something that would let you do

```yaml
policies:
  - resource: cloudfront
     filters:
        - type: config
           key: Logging.Enabled
           value: true
           op: not-equal
```

that covers the logging enabled but also allows access to any of the other distribution configuration options